docs/source/membership/membership.md

```diff
@@ -1,341 +1,194 @@
-identity belongs.
-
-We'll see later how OUs can be helpful to control the parts of an organization who
-are considered to be the members of a blockchain network. For example, only
-identities from the `ORG1-MANUFACTURING` OU might be able to access a channel,
-whereas `ORG1-DISTRIBUTION` cannot.
-
-Finally, though this is a slight misuse of OUs, they can sometimes be used by
-different organizations in a consortium to distinguish each other. In such cases, the
-different organizations use the same Root CAs and Intermediate CAs for their chain
-of trust, but assign the `OU` field to identify members of each organization.
-We'll also see how to configure MSPs to achieve this later.
-
-### Local and Channel MSPs
-
-MSPs appear in two types of places in a blockchain network:
+# Membership Service Provider (MSP)
+
+## Why do I need an MSP?
+
+Because Fabric is a permissioned network, blockchain participants need a way to prove their identity to the rest of the network in order to transact on the network. If you've read through the documentation on [identity](../identity/identity.html)
+you've seen how a Public Key Infrastructure (PKI) can provide verifiable identities through a chain of trust. How is that chain of trust used by the blockchain network? Recall from the credit card scenario in the Identity topic that the Certificate Authority is like a card
+provider — it dispenses many different types of verifiable identities. An MSP, on the other hand, determines which credit card providers are accepted at the store. In this way, the MSP turns an identity (the credit card) into a role (the ability to buy things at the store).
+
+This ability to turn verifiable identities into roles is fundamental to the way Fabric networks function, since it allows organizations, nodes, and channels the ability establish MSPs that determine who is allowed to do what at the organization, node, and channel level.
+
+![MSP1a](./membership.msp.diagram.png)
+
+*Identities are similar to your credit cards that are used to prove you can pay. The MSP is similar to the list of accepted credit cards.*
+
+Consider a consortium of banks that operate a blockchain network. Each bank operates peer and ordering nodes, and the peers endorse transactions submitted to the network. However, each bank would also have departments and account holders. The account holders would belong to each organization, but would not run nodes on the network. They would only interact with the system from their mobile or web application. So how does the network recognize and differentiate these identities? A CA was used to create the identities, but like the card example, those identities cant just be issued, they need to be recognized by the network. MSP's are used to define the organizations that are trusted by the network members. MSPs are also the mechanism that provide members with a set of roles and permissions within the network.
```

Comment:
"cant" should be "can't." Also should be MSPs, not "MSP's". 

I think this last sentence could be improved. You've gotten to the point where you've established MSPs conceptually and that they define an organization. The next logical step is to say that MSPs are also used to validate. Something like:

"Because the MSPs defining these organizations are known to the members of a network, they can then be used to validate that network entities that attempt to perform actions are allowed to."